hztb-servicemanager/src/main/java/com/svs/hztb/service/impl/GCMServiceImpl.java

```diff
@@ -15,7 +15,7 @@
 import com.svs.hztb.api.sm.model.opinion.RequestOpinionInput;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.svs.hztb.api.sm.model.opinion.RequestOpinionInput'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)